sigv4-signing/swift/sigv4/README.md

```diff
@@ -0,0 +1,51 @@
+# Sigv4 Signer Samples - Swift
+
+## Setup
+
+This sample application uses CocoaPods as its dependency manager.
+
+1. Install Cocoapods on your Mac if it isn't yet already installed.
+   
+```shell
+brew install cocoapods
+pod setup
+```
+
+2. Clone this repository and change directories to the directory containing this [`Podfile`](./Podfile):
+
+```shell
+git clone https://github.com/aws-samples/amazon-kinesis-video-streams-demos.git
+cd amazon-kinesis-video-streams-demos/sigv4-signing/swift/sigv4
+```
+
+3. Install the dependencies
+
+```shell
+pod install
+```
+
+4. Open the project using one of these two methods:
+
+   1. Open XCode, select "Open a project or file", and choose [sigv4.**xcworkspace**](sigv4.xcworkspace), **OR**
+  
+   2. Run the following command from the [sigv4](.) (this) folder.
+    ```bash
+    xed .
+    ```
+
+
+## Project Structure
+
+This project comes in 3 different parts:

```

Comment:
This sample is designed in a way that the customer application can be integrated using the Sigv4 signer logic in a Swift (iOS) application as a reference.

Reply:
Updated